test/prototype/test_smoothquant.py

```diff
@@ -0,0 +1,176 @@
+from copy import deepcopy
+import os
+import pytest
+import torch
+import tempfile
+from torch.ao.quantization import MovingAverageMinMaxObserver
```

Comment:
can you add a followup (e.g. open an issue to implement moving average observer in torchao?)

Reply:
Now it is not used in the implementation although using a MovingAverageMinMaxObserver might produce better accuracy. Do we still need a followup?

Reply:
is the change significant? if so maybe worth follow up

Reply:
Ok. I will open an issue then.